bombastic/model/src/data.rs

```diff
@@ -2,15 +2,15 @@ use std::fmt::Formatter;
 use tracing::{info_span, instrument};
 
 pub enum SBOM {
-    #[cfg(feature = "cyclonedx-bom")]
+    #[cfg(feature = "cyclonedx")]
     CycloneDX(cyclonedx_bom::prelude::Bom),
     #[cfg(feature = "spdx-rs")]
     SPDX(spdx_rs::models::SPDX),
 }
 
 #[derive(Debug, Default)]
 pub struct Error {
-    #[cfg(feature = "cyclonedx-bom")]
+    #[cfg(feature = "cyclonedx")]
```

Comment:
`cyclonedx-bom` is actually the name/ID of the crate. I think we should/have to keep that.

Reply:
Thanks for making a comment in this line. I admit I'm not very familiar with the `#[cfg` thing. I did tried to keep `cyclonedx-bom` but then this following line was never executed but always skipped; even the IDE alerted me that block of code will be disabled as the feature `cyclonedx` was disabled.

https://github.com/trustification/trustification/blob/b7c1230b113f6cf856c4b5a48f0071baff2715ae/bombastic/model/src/data.rs#L67

Are you sure this line must be `cyclonedx-bom` and not `cyclonedx` as it is currently in my PR? 
> This #[cfg] thing is a "must to learn" this week for me :)

Reply:
Done, thanks!